lib/Rules/Uppercase.php

```diff
@@ -23,19 +23,24 @@
 
 class Uppercase extends Base {
 
+	/**
+	 * @param $password
+	 * @param $val
+	 * @throws PolicyException
+	 */
 	public function verify($password, $val) {
 		if ($this->countCapitals($password) < $val) {
-			throw new \Exception(
-				$this->l10n->t("Password contains too few uppercase characters. Minimum %d uppercase characters are required.", [$val]));
+			throw new PolicyException(
+				$this->l10n->t('Password contains too few uppercase characters. Minimum %d uppercase characters are required.', [$val]));
```

Comment:
"_The password contains too few uppercase characters. A minimum of %d uppercase characters are required._"